src/v2/coins/algo.ts

```diff
@@ -0,0 +1,37 @@
+import BaseCoin = require('../baseCoin');
+import * as _ from 'lodash';
+
+
+class Algorand extends BaseCoin {
+
+    constructor() {
+      super();
+    }
+
+    getChain() {
+        return "algo";
```

Comment:
nit: use single quotes here and down in ln:16

Reply:
ack